google-ads-examples/src/main/java/com/google/ads/googleads/examples/campaignmanagement/ValidateAd.java

```diff
@@ -31,18 +33,19 @@
 import com.google.ads.googleads.v11.services.MutateAdGroupAdsRequest;
 import com.google.ads.googleads.v11.services.MutateAdGroupAdsResponse;
 import com.google.ads.googleads.v11.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.util.List;
 import java.util.stream.Collectors;
 
 /**
- * Shows how to use the validateOnly header to validate an expanded text ad. No ads will be created,
- * but exceptions will still be thrown.
+ * Shows how to use the validateOnly header to validate a responsive search ad. No ads will be
```

Comment:
I know the golden says "header" but I think they were thinking of the AdWords API where that was a header.
Could we say "validateOnly request parameter" instead?

Reply:
@BenRKarl FYI for Python because I took this language from that example.

Reply:
@jradcliff fixed. Leaving this open for Ben to see.